doc/events/task_complete.md

```diff
@@ -0,0 +1,26 @@
+---
+module: [kind=event] task_complete
+see: commands.execAsync To run a command which fires a task_complete event.
+---
+
+The @{task_complete} event is fired when a command run with @{commands.execAsync} or any function in @{commands.async} finishes.
```

Comment:
Hrmrmr.

_Technically_, this is fired on any asynchronous peripheral call (e.g. `drive.setDiskLabel`). However, most of the time it's invisible, as the peripheral call waits for the event.

I don't know if this behaviour is worth documenting or not. Definitely should drop the "This event is normally only available on Command Computers." statement as that's wrong, but the rest of the time it's really just an implementation detail.